### PR TITLE
Establish a centralized repository of location for Gisida to reference

### PR DESCRIPTION
**Pull request by kahama94**
fix: #297
Am pulling the location coordinates and zoom extents from a form in ona:
here is the link to the [enketo form ](https://enketo.ona.io/x/#whadRAos)based on country provided on the site config.

### CODE REVIEW

File: src/store/initStore.js [hunk 1]
```diff
@@ -81,6 +83,32 @@ function addConfigToStore(store, config) {
...
     store.dispatch(actions.initLoc(config.LOC));
   }
+  const { locFormId, country } = config.APP.mapConfig;
+  /**
+   * Check  location formId and country
+   * Get all locations and filter by country
+   */
+  if (country && locFormId) {
+    let q = d3.queue();
+    q = q.defer(getData, locFormId, null, config.APP);
```
Comment (KipSigei) [inline src/store/initStore.js]: @kahama94 can we refactor to use `fetch()` method instead of d3 when making requests?

**Review by KipSigei — CHANGES_REQUESTED** [review]

**Comment by KipSigei** [comment]
Lets also add tests for the newly created action and its reducer

File: src/store/reducers/reducers.js [hunk 2]
```diff
@@ -17,6 +17,18 @@ export function APP(state = defaultState.APP, action) {
   }
 }
 
+function LOCLIB(state = defaultState.LOCLIB, action) {
+  switch (action.type) {
+    case types.INIT_LOC_LIB:
+    {
+      console.log(action.config);
```
Comment (KipSigei) [inline src/store/reducers/reducers.js]: :ambulance:

Comment (kahummer) [reply src/store/reducers/reducers.js]: Cool will create issues to handle address the tests for this reducers

**Pull request closed without merging**